bin/tee-vault-unseal/src/main.rs

```diff
@@ -137,6 +141,14 @@ async fn main() -> Result<()> {
 
     let server_state = get_vault_status(&args.attestation.vault_addr, conn.client()).await;
 
+    // If sha file given, override env variable with contents
+    if let Some(sha_file) = args.vault_auth_tee_sha_file {
+        let mut file = std::fs::File::open(sha_file)?;
+        let mut contents = String::new();
+        file.read_to_string(&mut contents)?;
+        args.vault_auth_tee_sha = contents.trim_end().into();
+    }
+
```

Comment:
Nit: it maybe be bit unexpected if `--vault-auth-tee-sha=<SHA>` is set explicitly and `vault_auth_tee_sha_file` is set implicitly with the `VAULT_AUTH_TEE_SHA256_FILE` env variable, as it will overwrite `args.vault_auth_tee_sha` (if I understand correctly). I don't want to be too nit picky, so I approve.